setup.cfg

```diff
@@ -1,3 +1,17 @@
-[templer.local]
-template = plone_basic
+[isort]
+# for details see
+# https://docs.plone.org/develop/styleguide/python.html#grouping-and-sorting
+force_alphabetical_sort = True
+force_single_line = True
+lines_after_imports = 2
+line_length = 88
+multi_line_output=3
+include_trailing_comma=True
+not_skip = __init__.py
 
+[flake8]
+exclude = bootstrap.py,docs,*.egg.,omelette
+max-complexity = 15
+max-line-length = 88
```

Comment:
@ale-rt I would set line_length to something like 200 to not to collide with black.

Reply:
Black is 88

Reply:
Not always if I recall correctly. Do you have a pointer regarding this?

Reply:
https://github.com/psf/black#line-length

Reply:
@ale-rt right, I recall reading that and setting line_length to 88. Though, then at some point black was creating longer lines so we moved to 200.